easybuild/easyconfigs/t/Tk/Tk-8.6.1-ictce-5.3.0.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.1'
+
+homepage = 'http://www.tcl.tk/'
+description = """Tk is an open source, cross-platform widget toolchain 
+that provides a library of basic elements for building a graphical user 
+interface (GUI) in many different programming languages."""
+
+toolchain = {'name': 'ictce', 'version': '5.3.0'}
+
+source_urls = ["http://prdownloads.sourceforge.net/tcl"]
+sources = ['%s%s-src.tar.gz' % (name.lower(), version)]
+
+dependencies = [('Tcl', version)]
+
+configopts = '--enable-threads --with-tcl=$EBROOTTCL/lib CFLAGS="-I$EBROOTTCL/include"'
```

Comment:
this is there in all existing `Tk` easyconfigs, but it's probably not needed anymore, since `eb` will update `$CPATH` with this path in the `Tcl` module being loaded

the `--with-tcl` may still be needed, however

so, let's try to drop the `CFLAGS` part here, and then also in all other `Tk` easyconfig if it still works